src/wp-includes/class-wp-oembed.php

```diff
@@ -212,6 +210,8 @@ public function __construct() {
 		 * | Photobucket  | photobucket.com      | 2.9.0     | 5.1.0     |
 		 * | Funny or Die | funnyordie.com       | 3.0.0     | 5.1.0     |
 		 * | CollegeHumor | collegehumor.com     | 4.0.0     | 5.3.1     |
+		 * | CollegeHumor | collegehumor.com     | 4.0.0     | 5.3.1     |
```

Comment:
Now we have CollegeHumor twice in this list. We should remove this line.

Reply:
Sorry, that went unnoticed 😅